docs/credentials.rst

```diff
@@ -222,9 +222,8 @@ Select the Network credential type **only** if you are using a ``local`` connect
 - For ``local`` connections using ``provider``, credential type should be :guilabel:`Network`
 - For all other network connections (``httpapi``, ``netconf``, and ``network_cli``), credential type should be :guilabel:`Machine`
 
-For an overview of connection types available for network devices, refer to `Multiple Communication Protocols`_.
+For an overview of connection types available for network devices, refer to `Multiple Communication Protocols <https://docs.ansible.com/ansible/devel/network/getting_started/network_differences.html#multiple-communication-protocols>`_.
 
-  .. _`Multiple Communication Protocols`: https://docs.ansible.com/ansible/devel/network/getting_started/network_differences.html#multiple-communication-protocols.
```

Comment:
@tvo318 I think the problem here was that the trailing period was being parsed as a part of the URL. Just removing that dot would've fixed the problem.

Reply:
Ok - I just changed the format because I didn't like the number of lines it took up anyway.

Reply:
Detached refs are usually better because they don't interrupt plain text in paragraphs as much.

Reply:
OK, I'll do detached refs next time.